vtexdocs/RapiDoc
# Feature/section method path

#### What is the purpose of this pull request?

To update and style the documentation section that shows the corresponding REST method and path, as shown in the image below.
![image](https://user-images.githubusercontent.com/39717968/171057884-ca54a819-00c8-4322-a0db-139c4d21a50e.png)

[Further information here.](https://www.notion.so/vtexhandbook/Estilizar-Section-operation-url-RapiDoc-0f944339bc784d3f9cb6566254c57f6a)

#### What problem is this solving?

Inadequate structure and styling to follow [the expected design](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=3696%3A238901) for the section showing the REST method and the path referring to the documentation accessed by the user. Without changing the rapidoc source code as in this PR, other styling means such as CSS parts were not flexible enough to reach the desired look.

#### How should this be manually tested?

Check if there are any typos or styling errors in the added code. As this contribution has been given a beta version tag, you can view the achieved results in parallel with other changes that were applied to the [devportal](https://github.com/vtexdocs/devportal/pull/41) code (like removing style overrides) on [this deploy preview](https://deploy-preview-41--elated-hoover-5c29bf.netlify.app/).

#### Screenshots or example usage


https://user-images.githubusercontent.com/39717968/171062038-455780f9-878e-424b-80c3-d5406f6fe0b6.mp4



#### Types of changes
